lib/cgo/cipher.crypto.go

```diff
@@ -43,36 +43,6 @@ func SKY_cipher_PubKeyFromHex(_s string, _arg1 *C.cipher__PubKey) (____error_cod
 	return
 }
 
-//export SKY_cipher_PubKeyFromSecKey
```

Comment:
why is this removed ?

Reply:
Because to move https://github.com/simelo/libskycoin/blob/stdevAlDen_t7_refactor_in_tests/lib/cgo/cipher.crypto.hw.go#L139

Reply:
I see ...

Reply:
Revert that change . That makes it impossible to auto-generate code out of `skycoin/skycoin` function signatures.